xbmc/guilib/GUIFont.cpp

```diff
@@ -170,8 +177,10 @@ void CGUIFont::DrawScrollingText(float x,
                                  float maxWidth,
                                  const CScrollInfo& scrollInfo)
 {
-  if (!m_font) return;
-  if (!shadowColor) shadowColor = m_shadowColor;
+  if (!m_font)
+    return;
+  if (!shadowColor)
```

Comment:
My only comment on this PR would be that I would like to see some extra whitespace to help make it easier to read. In places like this it would be nice to have a newline after the return. There are quite a few places that could be updated but I'm only going to comment about this once 👍 Of course this is just my opinion and doesn't impact the functionality at all.

Reply:
Would be better, have added some new lines on places like this.